src/NUnitConsole/nunit-console.tests/TeamCityEventHandlerTests.cs

```diff
@@ -1,132 +0,0 @@
-﻿// ***********************************************************************
-// Copyright (c) 2015 Charlie Poole
-//
-// Permission is hereby granted, free of charge, to any person obtaining
-// a copy of this software and associated documentation files (the
-// "Software"), to deal in the Software without restriction, including
-// without limitation the rights to use, copy, modify, merge, publish,
-// distribute, sublicense, and/or sell copies of the Software, and to
-// permit persons to whom the Software is furnished to do so, subject to
-// the following conditions:
-// 
-// The above copyright notice and this permission notice shall be
-// included in all copies or substantial portions of the Software.
-// 
-// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
-// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
-// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
-// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
-// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
-// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
-// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-// ***********************************************************************
-
-using System;
-using System.IO;
-using System.Reflection;
-using System.Text;
-using System.Xml;
-using NUnit.Framework;
-
-namespace NUnit.ConsoleRunner.Tests
-{
-    using Utilities;
-
-    public class TeamCityEventHandlerTests
```

Comment:
You dropped all these tests and added entirely different tests. That means nobody is able to see, by looking at the changes in the test code, what you are doing differently. Instead, we must read the entire code to see what was done before and what is done now. That's a very large burden for the reviewer.

Is it possible for you to restore this test class and make the tests pass without much work? If it is redundant, then we can once again drop it after the review.